symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/MessageService.java

```diff
@@ -54,6 +137,58 @@ public V4Message send(String streamId, String message) {
     }
   }
 
+  /**
+   * Retrieved the details of a message given its message ID.
+   *
+   * @param messageId the ID of the message to be retrieved
+   * @return a {@link V4Message} containing the message's details, null if the message was not found
+   * @see <a href="https://developers.symphony.com/restapi/reference#get-message-v1">Get Message v1</a>
+   */
+  public V4Message getMessage(@Nonnull String messageId) {
+    try {
+      return this.messagesApi.v1MessageIdGet(authSession.getSessionToken(), authSession.getKeyManagerToken(),
+          messageId);
+    } catch (ApiException e) {
+      log.error("Cannot get message", e);
+      sleep(1_000);
```

Comment:
why do we sleep here?

Reply:
@symphony-elias could you remove this ugly retries :) Remember that we have a dedicated story for retry configuration and implementation (APP-2821). At the moment, just throw a `new ApiRuntimeException()`